introduction.md

```diff
@@ -1,3 +1,3 @@
 Hi there.  My favorite Code school course is Javascript.
 
-* Git, Javascript
\ No newline at end of file
+* Git, Javascript, CSS
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?